### PR TITLE
Week3 step5

### PR DESCRIPTION
**Pull request by Xin128**
Step 5 finished:

Connect to the database from DatastoreService

### CODE REVIEW

**Comment by cisxingg** [comment]
Need to add the "wait on reviews before merging" setting

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java
Comment (cisxingg) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: Spacing: https://google.github.io/styleguide/javaguide.html#s3.3.3-import-ordering-and-spacing

**Review by cisxingg — APPROVED** [review]
LG, just one nit :)

**Comment by Xin128** [comment]
Hi Cis,
As this branch is just merging to branch Week3Step4, do I also need to add the "need to merge" settings? If I merge to master, it needs the blocking for approval.  I can definitely do that, but I am wondering if this will be kind of redundant as we probably will create a lot of branches later.

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java
Comment (Xin128) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: Just to double check: should I add the declaration of ArrayList<String> on line 46 instead of putting a new empty line in between. I just thought from line 47 to line 52 are doing one blocked feature

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java
Comment (warfa-google) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: Save this as a constant at the top of the file. Try to stay away from hardcoded values in code

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java
Comment (warfa-google) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: Use the comment constant from the above comment here

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java
Comment (warfa-google) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: also make "content" a constant

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java
Comment (warfa-google) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: make this final private and by convention should be uppercase.

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java
Comment (warfa-google) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: make "content" a global constant

**Review by warfa-google — APPROVED** [review]
Fix small constant comments but otherwise lgtm

**Pull request merged**
